assets/js/print.mjs

```diff
@@ -0,0 +1,28 @@
+const initPrintLinks = () => {
```

Comment:
This bit of JS is required to support print.
Basically, where we have pages that have the govUk details expander, when the user prints the page we need the expanders to be opened, so that their contents is included in the print out.

This file was originally written for the same reason for LWP, and I simply lifted it from there 😁